File: argonaut-spring-ai/src/main/java/dev/jsanca/argonaut/springai/api/AboutController.java
package dev.jsanca.argonaut.springai.api;

import dev.jsanca.argonaut.core.experiment.ArgonautInfo;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

@RestController
@RequestMapping("/api")
public class AboutController {

    @GetMapping("/about")
    public ArgonautInfo about() {
        return new ArgonautInfo(
                "spring-ai",
                "Spring AI",
                "2.0.0",
                "OpenRouter",
                null,
                List.of("controlled-local-evidence-rag", "tool-calling", "observability"));
    }
}

File: argonaut-koog/src/main/kotlin/dev/jsanca/argonaut/koog/api/AboutController.kt
package dev.jsanca.argonaut.koog.api

import dev.jsanca.argonaut.core.experiment.ArgonautInfo
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api")
class AboutController {

    @GetMapping("/about")
    fun about(): ArgonautInfo = ArgonautInfo(
        "koog",
        "Koog",
        "1.1.1",
        "OpenRouter",
        null,
        listOf("UC-001: Controlled Local Evidence RAG")
    )
}
